modules/network-observability-operator-install.adoc

```diff
@@ -9,7 +9,7 @@ You can install the Network Observability Operator using the {product-title} web
 
 .Prerequisites
 
-* Installed Loki. It is recommended to install Loki using the link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.7].
+* If you choose to use Loki, install the link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.7].
```

Comment:
Link opens to version `v5.7.5-5`, which prvoides a `.z` granularity to versioning. Does this matter?

Reply:
I don't think so